source/_integrations/rainbird.markdown

```diff
@@ -126,12 +128,11 @@ The service can be used as part of an automation script. For example:
 automation:
   - alias: "Turn irrigation on"
     trigger:
-      platform: time
-      at: "5:30:00"
+      - platform: time
+        at: "5:30:00"
     action:
-      service: rainbird.start_irrigation
-      target:
-        entity_id: switch.sprinkler_1
-      data:
-        duration: 5
+      - service: rainbird.start_irrigation
+        data:
+          entity_id: switch.sprinkler_1
+          duration: 5
```

Comment:
This is actually incorrect / a bug in the rainbird integration.
The service here is registered incorrectly in the code base.

I'm not sure if we should "workaround" in documentation or open an issue (or better) a PR to fix it.

Reply:
Oh interesting.  This is the values generated by the automation config UI and it seemed to work (though I'll admit I didn't let it run the full 5 minutes and stopped it with the rainbird app)

Reply:
Ooh this change  you are sending in works...
but the issue is that is should work with a target :)